stages/org.osbuild.bootc.install-to-filesystem

```diff
@@ -12,6 +13,19 @@ def main(options, inputs, paths):
     assert len(images) == 1
     image = list(images.values())[0]
 
+    env = os.environ.copy()
+    # By default "bootc" will refuse to work on a non-selinux system if
+    # the bootc container requires selinux. This is a sensible approach
+    # in general but for us it's tricky because we want to be able to
+    # generate images when running on developer machines or CI machines
+    # that may not necessarily have selinux. So make bootc more relaxed.
+    env["BOOTC_SKIP_SELINUX_HOST_CHECK"] = "true"
```

Comment:
I don't like doing this unconditionally; can we tell it through the schema?

Reply:
Yeah, it's not great, I agree. I was thinking about this too (assuming you mean a schema value like "skip-selinux-host-check: bool" that then would be put into the manifest):
This seems not an ideal place because it feels a bit odd that the manifest.json would be dependent on what host osbuild runs on. Also the schema could be generated on a different machine, it could be generated a long time ago on a host with selinux and then it got disabled and suddently the manifest stops works.

To make it conditional we would have to find a way to pass it from osbuild to the stage without a schema/manifest (IMO), I don't recall we have a mechanism today that can do that?

But yeah, if you/Colin feels strongly about this we need to make it conditional.

Reply:
> but for us it's tricky because we want to be able to generate images when running on developer machines or CI machines that may not necessarily have selinux.

Handling it for CI is trivial, set the environment variable in the CI flow which is exactly what bootc is doing for its CI.

I wouldn't really distinguish "developer" from "user" in general here - having it on by default here  I think this tantamount to supporting it in practice.  So maybe we should just do that.

If it works for you now, then yeah we can just go ahead and change bootc and claim to support it and just try to keep testing it.

I know for sure we need to support it longer term so it may be the right time to just say we are, and if it breaks we fix it.

Reply:
Does this affect the output image?

Reply:
No effect on output.